index.html

```diff
@@ -11,272 +11,227 @@
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">css</li>
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
+              </ul>
+              <a href="" class="btn-a w-part">See Project</a>
+            </article>
+            <article class="work-item">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
+              </ul>
+              <a href="" class="btn-a">See Project</a>
+            </article>
+            <article class="work-item">
+              <h1>Profesional Art Printing Data</h1>
+              <p class="inter">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
+              </ul>
+              <a href="" class="btn-a">See Project</a>
+            </article>
+            <article class="work-item">
+              <h1>Profesional Art Printing Data</h1>
+              <p class="inter">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
+              </ul>
+              <a href="" class="btn-a">See Project</a>
+            </article>
+          </div>
+        </section>
+        <section class="full-h bkg-black" id="about-me">
+          <div class="flex-all">
+            <h1 class="crete">About Me</h1>
+            <p class="poppins">
+              I can help you build a product, feature or website. Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don't hesitate to contact me.
+            </p>
+            <a href="" class="btn-a w-part">Get My Resume</a>
+          </div>
+          <hr class="pad-right" />
```

Comment:
- [ ] Good job for making this part look awesome 👏 However, there are still some details that need to be worked on 😅 
- [ ] Kindly add the right `font-weight` to the title "About Me" 👍 
- [ ] Kindly reduce the bottom margin of the title to match the design 👍 
- [ ] Kindly increase the height of the about me section or add padding bottom to it to match the design in Figma 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/212332096-eccaa4fa-debb-462d-84ab-2a0557e2b682.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/212332169-d2a18ab1-d98a-4c6a-bb8e-43e62aa1574d.png)